articles/100_ament.md

```diff
@@ -20,34 +20,93 @@ author: '[Dirk Thomas](https://github.com/dirk-thomas)'
 Original Author: {{ page.author }}
 
 
+## Preface
+
+ROS is developed with a federated model.
+While this comes with a lot of advantages it makes building a set of interdependent packages more complex.
+
+Commonly a user has to build interdependent packages in the correct topological order.
+And for each package the steps include reading the documentation of a package, then building and installing it following a few build system specific commands and then setting up the environment for other packages to use it.
```

Comment:
Then again, my fifth grade teacher was kind of pedantic, you can take or leave any part of the suggested text.